wp-content/themes/pub/wporg-learn-2024/src/style/_sensei.scss

```diff
@@ -26,26 +53,153 @@ body.sensei {
 		display: none;
 	}
 
+	// Facilitator notes
+	#wporg-learn-facilitator-notes-toggle {
+		display: none;
+
+		&:checked + .wporg-learn-facilitator-notes-label + .wporg-learn-facilitator-notes-content {
+			display: block;
+			position: absolute;
+			top: calc(var(--sensei-lm-header-height));
+			right: 0;
+			width: 50%;
+			height: 100vh;
+			margin-top: 1px;
+			border-left: 1px solid var(--sensei-course-progress-bar-color);
+
+			.wporg-learn-facilitator-notes-close {
+				background: url(../../assets/icon-close.svg);
+				background-repeat: no-repeat;
+				background-position: center;
+				width: 24px;
+				height: 24px;
+				position: absolute;
+				right: var(--wp--preset--spacing--40);
+				top: calc(var(--wp--preset--spacing--30) + var(--wp--preset--spacing--20));
+				cursor: pointer;
+			}
+
+			@media (max-width: 782px) {
+				position: absolute;
+				top: var(--wp--preset--spacing--10);
+				left: 4%;
+				width: 92%;
+				height: 90vh;
+				border: 1px solid var(--sensei-course-progress-bar-color);
```

Comment:
Seeing a grey border around the modal on mobile, I don't think this matches modals on other sites, eg. Meeting Calendar

| Facilitator notes | Facilitator notes inspected | Meeting Calendar |
|--------|--------|-|
| ![Screenshot 2024-10-17 at 11 36 15 AM](https://github.com/user-attachments/assets/8a3fae34-670a-4478-9a9a-8c24d06cc122) | ![Screenshot 2024-10-17 at 11 36 24 AM](https://github.com/user-attachments/assets/a3f82a1a-3484-4580-8c1a-30f0d9b32304) | ![Screenshot 2024-10-17 at 11 34 56 AM](https://github.com/user-attachments/assets/4ed97e69-c9eb-4474-b976-6bb512da2dfe) |